docs/pages/app-signing/app-credentials.mdx

```diff
@@ -73,3 +74,18 @@ Provisioning profiles expire after 12 months, but this won't affect apps in prod
 ### Clearing Credentials
 
 When you use the `eas credentials` command to delete your credentials, this only removes those credentials from Expo's servers, **it does not delete the credentials from Apple's perspective**. This means that to fully delete your credentials (for example, if you want a new push notification key, however, you already have two), you'll need to do so from the [Apple Developer Console](https://developer.apple.com/account/resources/certificates/list).
+
+### Resigning new credentials
+
+You can use `eas build:resign` to codesign an existing **.ipa** for iOS to a new ad hoc provisioning profile. This helps reduce time when distributing internally, for example, when you have multiple test iOS devices or want to share a build with your team members.
+
+Running the command will ask you to select a build that you want to resign. For example, running the command in an example project shows an available build:
```

Comment:
```suggestion
Running the command will ask you to select a build that you want to re-sign. For example, running the command in an example project shows an available build:
```

it's re-sign in a sentence but resign in a command. the convention is to remove - in command names